doc/kubernetes/ray-cluster-external-redis.yaml

```
@@ -0,0 +1,136 @@
+# Ray head node service, allowing worker pods to discover the head node.
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: ray
+  name: example-cluster-ray-head
+spec:
+  ports:
+  - name: client
+    protocol: TCP
+    port: 10001
+    targetPort: 10001
+  - name: dashboard
+    protocol: TCP
+    port: 8265
+    targetPort: 8265
+  - name: redis
+    protocol: TCP
+    port: 6379
```

The port for the GCS server should ideally not conflict with the external Redis's port, even if GCS and Redis run at different IPs. Just for sanity, it would be good to use, say, 6380.
```suggestion
    port: 6380
```